train.py

```diff
@@ -90,6 +90,10 @@ def train(hyp, opt, device, callbacks):  # hyp is path/to/hyp.yaml or hyp dictio
     data_dict = None
     if RANK in {-1, 0}:
         loggers = Loggers(save_dir, weights, opt, hyp, LOGGER)  # loggers instance
+        if loggers.clearml:
+            # Data_dict is either None is user did not choose for ClearML dataset or is filled in by ClearML
+            data_dict = loggers.clearml.data_dict
+
```

Comment:
@thepycoder I didn't understand this part. If `loggers.clearml.data_dict`is None when users don't use clearml dataset management, then it'd make data_dict None which would break other parts right?

Reply:
You're right it would make `data_dict` None, but that's the goal. A little more down (L110), the original code checks if `data_dict` is set and if not, uses `check_dataset` to fill it. The None is basically just saying: it was not filled by any third party, so fallback to standard yolo dataset

```
    with torch_distributed_zero_first(LOCAL_RANK):
        data_dict = data_dict or check_dataset(data)  # check if None 
```

Reply:
Thanks!